core/sql/regress/seabase/EXPECTED010

```diff
@@ -10316,12 +10319,12 @@ CREATE TABLE TRAFODION.SCH."delim_tab"
 --- SQL operation complete.
 >>get all volatile schemas;
 
-Schema(Active  ): VOLATILE_SCHEMA_MXID110000212952122994710660444710000000002
+Schema(Active  ): VOLATILE_SCHEMA_MXID110000212132123073588113805260000000002
 
 --- SQL operation complete.
 >>get all volatile tables;
 
-Schema(Active  ): VOLATILE_SCHEMA_MXID110000212952122994710660444710000000002
+Schema(Active  ): VOLATILE_SCHEMA_MXID110000212132123073588113805260000000002
   Table: VTAB1
   Table: VTAB2
 
```

Comment:
That sounds a good idea.

How about I added some next time?

On Thu, Aug 27, 2015 at 11:07 AM, Suresh Subbiah <notifications@github.com>
wrote:

> In core/sql/regress/seabase/EXPECTED010
> <https://github.com/apache/incubator-trafodion/pull/66#discussion_r38113428>
> :
>
> >
> >  --- SQL operation complete.
> >  >>get all volatile tables;
> >
> > -Schema(Active  ): VOLATILE_SCHEMA_MXID110000212952122994710660444710000000002
> > +Schema(Active  ): VOLATILE_SCHEMA_MXID110000212132123073588113805260000000002
> >    Table: VTAB1
> >    Table: VTAB2
> >
>
> i am guilty of not doing this many times. It will be good to add a query
> or two to regression tests that will show the end-key being added due to
> this change. This is an important change and also one that could be
> unintentionally broken by future work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/66/files#r38113428>.
>



-- 
Regards, --Qifan